examples/SimpleClient.hs

```diff
@@ -24,7 +26,7 @@ main = do
           mErr1 <- executeDatabaseContextExpr sessionId conn (Define "person" (map NakedAttributeExpr attrList))
           print mErr1
           --5. add a tuple to the relation referenced by the relation variable
-          let (Right tupSet) = mkTupleSetFromList attrs [[TextAtom "Bob", IntAtom 45]]
+          let tupSet = fromRight undefined $ mkTupleSetFromList attrs [[TextAtom "Bob", IntAtom 45]]
```

Comment:
This doesn't seem like an improvement. Previously, the error would not that it is a pattern match failure. Now you'd get whatever `undefined` emits...

Reply:
The motivation here was also to suppress the "incomplete pattern match" warning. The solution I inserted is admittedly dubious and less clear than the original code, made in haste to quickly suppress the error. Given your suggestion to use a module specific `ghc` option flag in the `examples/Plantfarm.hs` module, I have reverted these changes and instead suppressed the warning with a `OPTIONS_GHC` pragma.